lunatrace/bsl/hasura/migrations/lunatrace/1652400983913_add-pr-comment-disable/up.sql

```diff
@@ -1,60 +1,14 @@
-ALTER TABLE public.settings ADD COLUMN pr_feedback_enabled boolean DEFAULT true NOT NULL;
 ALTER TABLE public.settings DROP COLUMN is_org_settings;
-ALTER TABLE public.projects ADD CONSTRAINT project_settings_fkey FOREIGN KEY (settings_id) REFERENCES public.settings;
-ALTER TABLE public.organizations ADD CONSTRAINT organization_settings_fkey FOREIGN KEY (settings_id) REFERENCES public.settings;
+ALTER TABLE projects DROP COLUMN settings_id;
+ALTER TABLE organizations DROP COLUMN settings_id;
 
---  get caught up on the missing settings rows
-DO $$
-    declare
-        project record;
-    BEGIN
-FOR project IN SELECT * FROM public.projects WHERE settings_id IS NULL
-LOOP
-    WITH setting AS (INSERT INTO public.settings DEFAULT VALUES RETURNING id)
-    UPDATE public.projects
-    SET settings_id=setting.id
-    FROM setting
-    WHERE public.projects.id = project.id;
-END LOOP;
-END; $$;
+ALTER TABLE public.settings ADD COLUMN pr_feedback_disabled boolean DEFAULT false;
 
-DO $$
-    declare
-        organization record;
-    BEGIN
-        FOR organization IN SELECT * FROM public.organizations WHERE settings_id IS NULL
-            LOOP
-                WITH setting AS (INSERT INTO public.settings DEFAULT VALUES RETURNING id)
-                UPDATE public.organizations
-                SET settings_id=setting.id
-                FROM setting
-                WHERE public.organizations.id = organization.id;
-            END LOOP;
-    END; $$;
+ALTER TABLE public.settings ADD COLUMN "project_id" uuid null;
```

Comment:
project reference is now a back reference instead of a forward reference. this lets us upsert into the settings table from a project page.

Reply:
The reason it was the other way is because there is a polymorphic relation from <Table> -> settings.  If you want to build a polymorphic relation with the foreign key in the child table you have to do something more complex. 

It's maybe not worth having the polymorphic relationship since the settings aren't exactly 1:1 anyway. The idea was that most of the columns would be overlapping since org level settings are mainly for overriding/bulk setting project settings.  That was the idea. Maybe too-smart-by-half but we've lost that ability here.